app/controllers/api/functions.php

```diff
@@ -191,11 +187,7 @@
     ->param('range', '30d', new WhiteList(['24h', '7d', '30d', '90d']), 'Date range.', true)
     ->inject('response')
     ->inject('dbForProject')
-    ->action(function ($functionId, $range, $response, $dbForProject) {
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Document $project */
-        /** @var Utopia\Database\Database $dbForProject */
-        /** @var Utopia\Registry\Registry $register */
+    ->action(function (string $functionId, ?string $range, Response $response, Database $dbForProject) {
```

Comment:
Not sure if `?string` makes sense here?

Since it will never be `null` and is always set to `30d`.

Reply:
I think as long as it's set as optional we should be strict and keep it as is.

Reply:
But it's never optional here at this point, it will always be what is set as default in the param.